app/controllers/admin/depositors_search_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller to allow searching for depositor SUNet IDs given druids
+  class DepositorsSearchController < Admin::ApplicationController
+    def index
+      authorize!
+
+      @depositors_search_form = Admin::DepositorsSearchForm.new(**depositors_search_params)
+
+      if @depositors_search_form.valid?
+        @druid_to_depositor_sunetid_map = Admin::DepositorsSearch.call(form: @depositors_search_form)
+        render :index, status: :created
+      else
+        render :new, status: :unprocessable_entity
+      end
```

Comment:
Is it weird for `render :index` to need `status: :created` (instead of defaulting to `status: :ok`)? Because I found that this was needed, probably to escape some underlying behavior. Maybe turbo-related? I'm not sure at the moment.